internal/cloudinit/cloudinit.go

```diff
@@ -484,6 +484,10 @@ Invoke-WebRequest -Uri "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{
 New-NetFirewallRule -DisplayName "ALLOW TCP PORT 9079" -Direction inbound -Profile Any -Action Allow -LocalPort 9079 -Protocol TCP
 Start-Process -FilePath "C:\Program Files\lite-engine\lite-engine.exe" -ArgumentList "server --env-file=` + "`" + `"C:\Program Files\lite-engine\.env` + "`" + `"" -RedirectStandardOutput "{{ .LiteEngineLogsPath }}" -RedirectStandardError "C:\Program Files\lite-engine\log.err"
 
+if ($env:COMPUTERNAME -eq "PACKER-63BBB9DE") {
```

Comment:
we just want to add this for our hosted vms. Since we don't pass any flag while we create the vm, im using COMPUTERNAME env variable to recognise it. This is same across all the windows hosted vms.